about/about-networking.md

```diff
@@ -0,0 +1,152 @@
+---
+title: About Networking
+description: Learn about networking!
+---
+
+You can get up and running with Calico by following any of our getting started guides without needing to be a networking
+expert. Calico hides the complexities for you.  However, if you would like to learn more about networking so you can
+better understand what is happening under the covers, this guide provides a short introduction to some of the key
+fundamental networking concepts for anyone who is not already familiar with them.
+
+In this guide you will learn:
+- The terms used to described different layers of the network.
+- The anatomy of a network packet.
+- What MTU is and why it makes a difference.
+- How IP addressing, subnets, and IP routing works.
+- What an overlay network is.
+- What DNS and NAT are.
+
+### Network layers
+
+The process of sending and receiving data over a network is commonly categorized into 7 layers (referred to as the {%
+include open-new-window.html text='OSI model' url='https://en.wikipedia.org/wiki/OSI_model' %}). They layers are
+typically abbreviated as L1 - L7. You can think of data as passing through each of these layers in turn as it is sent or
+received from an application, which each layer being responsible for a particular part of the processing required to
+send the data over the network.
+
+![OSI network layers diagram]({{site.baseurl}}/images/osi-network-layers.svg)
+
+In a modern enterprise or public cloud network, the layers commonly map as follows:
+
+- L5-7: all the protocols most application developers are familiar with. e.g. HTTP, FTP, SSH, SSL, DNS.
+- L4: TCP or UDP, including source and destination ports.
+- L3: IP packets and IP routing.
+- L2: Ethernet packets and Ethernet switching.
+
+### Anatomy of a network packet
+
+When sending data over the network, each layer in the network stack adds its own header containing the control/metadata
+the layer needs in order to process the packet as it traverses the network, passing the resulting packet on to the next
+layer of the stack. In this way the complete packet is produced, which includes all the control/metadata required by
+every layer of the stack, without any layer understanding the data or needing to process the control/metadata of
+adjacent network layers. 
+
+![Anatomy of a network packet]({{site.baseurl}}/images/anatomy-of-a-packet.svg)
+
+### MTU
```

Comment:
I'd still like to move this to the bottom.  It's quite an advanced topic - is this really the third thing you'd like to tell people about about networking.